app/models/open_library.rb

```diff
@@ -0,0 +1,2 @@
+class OpenLibrary < ApplicationRecord
+end
```

Comment:
What's the use for this?

Reply:
if I don't add this worpaginate it will break